firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/ReleaseIdentificationUtils.java

```diff
@@ -29,7 +29,7 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipFile;
 
-public final class ReleaseIdentificationUtils {
+final class ReleaseIdentificationUtils {
   private static final String TAG = "ReleaseIdentification";
   private static final int BYTES_IN_LONG = 8;
 
```

Comment:
I think the methods in this class should also be package-private

Reply:
Good call. I thought since the class is package-private it was fine but its better to be consistent.